hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -568,9 +568,20 @@ protected long acquireLock() throws UnknownHostException, TException, Interrupte
             Lists.newArrayList(lockComponent),
             System.getProperty("user.name"),
             InetAddress.getLocalHost().getHostName());
+    LOG.warn(
```

Comment:
Nit: maybe add timestamp for lock method?

Reply:
same as above.